src/cdrizzleapi.c

```diff
@@ -23,425 +23,447 @@
 static PyObject *gl_Error;
 FILE *driz_log_handle = NULL;
 
-/** --------------------------------------------------------------------------------------------------
+/** ---------------------------------------------------------------------------
  * Multiply each pixel in an image by a scale factor
```

Comment:
Isn't this type of comment used for autogeneration of documentation?  If so, why not use the tags associated with that like 'at' brief, 'at' param, etc?